packages/swirl-components/src/components/flip-lightbox/flip-lightbox.css

```diff
@@ -0,0 +1,146 @@
+:host {
+  display: block;
+
+  & * {
+    box-sizing: border-box;
+  }
+}
+
+.lightbox {
+  position: fixed;
+  z-index: var(--s-z-40);
+  background-color: rgba(0, 0, 0, 1);
+  inset: 0;
+
+  &[aria-hidden="true"] {
+    display: none;
+  }
+
+  @media (--from-tablet) {
+    background-color: rgba(0, 0, 0, 0.6);
+  }
+}
+
+.lightbox:not(.lightbox--closing) {
+  & .lightbox__body {
+    animation: 0.15s lightbox-fade-in;
+
+    @media (prefers-reduced-motion) {
+      animation: none;
+    }
+  }
+}
+
+.lightbox--closing {
+  animation: 0.15s lightbox-fade-out;
+  animation-fill-mode: forwards;
+
+  @media (prefers-reduced-motion) {
+    animation: none;
+  }
+}
+
+.lightbox__body {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.lightbox__header {
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  padding: var(--s-space-12) var(--s-space-16);
+  justify-content: space-between;
+  align-items: flex-start;
+  pointer-events: none;
+}
+
+.lightbox__controls {
```

Comment:
**issue (blocking):** i cannot see this part

Reply:
Yep. Maybe I screwed things up during merge.